python/lsst/ip/isr/defects.py

```diff
@@ -185,6 +185,10 @@ def _normalize(self):
         if self._bulk_update:
             return
 
+        # If we have no defects, there is nothing to normalize.
+        if len(self) <= 0:
```

Comment:
can `len` return a negative number?

Reply:
python is smarter than I thought, and even if you maliciously craft a class, you cannot return a negative length.  Updated to "==".